### PR TITLE
Fix Images of Pod Members in All Screen Sizes (Profile Section)

### PR DESCRIPTION
**Pull request by Swatilekha-Roy**
- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

@khattakdev 
- @Devansh3712 and I have worked on the image gallery and made a 3-D carousel with the images of all the pod members. 
- We have also fixed some other styling issues like placement of the Pod Number and headings.
- We have also made the site responsive for small screen sizes.

Hope this addresses our follow ups to Issue #15! Please feel free to make any suggestions for improvement or changes, if any, so that we can work on them. Thank you!

Screenshot (406)

### CODE REVIEW

**Comment by atharmohammad** [comment]
like this :
carousel

**Review by atharmohammad — APPROVED** [review]

**Comment by khattakdev** [comment]
@Swatilekha-Roy @Devansh3712 This looks really really cool

Can you please resolve the conflicts and also did you reduce the images' opacity? I can see the images in the back
image

File: _includes/header.html [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
-        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
-        integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
+  <!-- Custom CSS -->
+  <link lang="sass" rel="stylesheet" href="/assets/css/main.css" />
+  <link rel="icon" href="/assets/img/favicon.ico" type="image/x-icon" />
 
-    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
-    integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
-    crossorigin=""/>
```
Comment (atharmohammad) [inline _includes/header.html:31]: this should be in header else map will not appear

File: _includes/header.html [hunk 2]
```diff
@@ -1,40 +1,55 @@
...
+  ></script>
 
-    <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
-    integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
-    crossorigin=""></script>
-
-    <link lang='sass' rel="stylesheet" href="/assets/css/main.css">
-    <link rel='icon' href='/assets/img/favicon.ico' type='image/x-icon' />
-    <title>{{ site.title }}</title>
-</head>
```
Comment (atharmohammad) [inline _includes/header.html:40]: include this also

**Review by atharmohammad — CHANGES_REQUESTED** [review]
Please make these changes and check whether map is rendering or not .

File: _includes/header.html (hunk 2 above)
Comment (Swatilekha-Roy) [reply _includes/header.html]: @atharmohammad thanks so much for pointing out that major merging error on my part - totally my bad! Thanks so much. I have re-added the links and the map shows up just okay now! Thanks again. Please feel free to add in any other suggestions that you have.

File: _sass/all.scss [hunk 3]
```diff
@@ -39,12 +39,13 @@ body {
   margin-bottom: 50px;
   position: relative;
 }
+
 h2 {
-  margin-bottom: 20px;
-  margin-top: 20px;
-  font-weight: 700;
-  text-align: center;
-  color: var(--color-p-text);
```
Comment (atharmohammad) [inline _sass/all.scss:47]: in dark theme , color of h2 headings are not changing. you need to include this for it

**Comment by khattakdev** [comment]
I can see a small big in the footer (color of **IMPORTANT LINKS** )
image

But that's fine for now. This PR has already a lot of code changes.

**Pull request merged**